openedx/core/djangoapps/notifications/templates/notifications/email_digest_preference_update.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>{{ _("Email Digest Preferences Updated") }}</title>
+    </head>
+    <body>
+        <script>
+            alert('{{ _("You have successfully unsubscribed from email digest for learning activity") }}');
+            window.location.replace("{{ notification_preferences_url }}");
```

Comment:
This practice is not recommended. Please discuss it with the team.

Reply:
As discussed, we will update the solution once we get mockups.